opal/mca/common/cuda/common_cuda.c

```diff
@@ -1691,9 +1691,9 @@ static void cuda_dump_memhandle(int verbose, void *memHandle, char *str) {
     }
     memcpy(&memH, memHandle, sizeof(memH));
     opal_output_verbose(verbose, mca_common_cuda_output,
-                        "%s:ctxId=%d, pid=%d, size=%d, blocksize=%d, offset=%d, gpuId=%d, "
+                        "%s:ctxId=%p, pid=%d, size=%d, blocksize=%d, offset=%d, gpuId=%d, "
```

Comment:
Do you really want `%p` here?  `ctxId` is a `uint64_t`, so don't you want `PRIu64`, or `PRIx64`?